.github/workflows/deploy-dev.yml

```diff
@@ -55,7 +55,8 @@ jobs:
           IPFS_PROJECT_SECRET: '${{ secrets.IPFS_PROJECT_SECRET }}'
           FAUCET_SERVER_URL: 'https://faucet-dev.hashed.systems'
           GATED_PALLET_ID: '0x5d208df71427902157988a56e7527474a7dce921f9d5efa159dcfc849289a53c'
-          GATED_MARKETPLACE_ID: '0x1feda8a89ef968df5fbe082e4c48f88eed4dcb0ec11ca12d033e963e72594873'
+          GATED_MARKETPLACE_ID: '0xb3914e1762ff9ca8caec57959715350885632a8a323b8c160fc814600669ad33'
```

Comment:
This is a remote marketplace id???

Reply:
Is the ID of the M3 of Afloat. It is used to point to the marketplace that is for Afloat [Afloat Tab].

So with this ID, the UI gets the information of the Afloat Marketplace without getting others' market info

@jmgayosso